lib/rubocop/ast/node/mixin/wrapped_arguments_node.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # Common functionality for nodes that may have their arguments
+    # wrapped in a `begin` node
+    module WrappedArgumentsNode
+      # Returns the arguments of the `return`.
```

Comment:
I guess we shouldn't mention `return` here explicitly.

Reply:
Oops, not sure why I "duplicated" this comment...

Reply:
Oh, I know, I copy-pasted it as it was written 😅